virtuoso.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+SETTINGS_DIR=/settings
```

Comment:
I like having it in a var to prevent typo's, especially since I might reuse it for other guard files like .dba_pwd_set